include/handler.h

```diff
@@ -274,6 +278,12 @@ namespace detail {
 
 } // namespace detail
 
+/// Pass to `handler::host_task` to select the master-node task overload.
+inline constexpr detail::on_master_node_tag on_master_node;
+
+/// Equivalent to `range<0>{}` when passed to `handler::host_task`.
```

Comment:
Maybe document explicitly that this is used to execute the associated code only once on some node of the cluster.
That is implicit for us from what is currently written, but since this is user-facing documentation it might make things more clear.

Edit: just saw that this is documented at the function call level anyway.